app/controllers/api/v1/bracks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::BracksController < ApplicationController
+
+  def index
+    latlng = params[:latlng]
```

Comment:
I use it in two lines! I need to pass it in to both sort by distance, and also to add the distances to the serializer. Let me know if you think I shouldn't set it to a local var still.

```
latlng = params[:latlng]
@bracks = Brack.sort_by_distance(latlng)
render json: @bracks, latlng: latlng
```